wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientListViewController.swift

```diff
@@ -207,11 +207,14 @@ final class ClientListViewController: UIViewController,
             contextProvider: contextProvider
         )
         viewModel.showCertificateUpdateSuccess = {[weak self] certificateChain in
-            self?.updateAllClients {
-                self?.updateE2EIdentityCertificateInDetailsView()
+            guard let self = self, let lastE2EIUpdateDateRepository = userSession.lastE2EIUpdateDateRepository else {
```

Comment:
`lastE2EIUpdateDateRepository` doesn't seem to be used, please remove it.

Reply:
done